services-gateway-tests/src/test/java/io/scalecube/services/gateway/rsocket/RSocketLocalGatewayTest.java

```diff
@@ -9,11 +9,13 @@
 import io.scalecube.services.examples.GreetingResponse;
 import io.scalecube.services.examples.GreetingService;
 import io.scalecube.services.examples.GreetingServiceImpl;
+import io.scalecube.services.exceptions.InternalServiceException;
 import io.scalecube.services.gateway.BaseTest;
 import java.time.Duration;
 import java.util.List;
 import java.util.stream.Collectors;
 import java.util.stream.IntStream;
+import javax.naming.ServiceUnavailableException;
```

Comment:
Noticed here -- `javax.naming.ServiceUnavailableException`, didn't check rest of the code. Plz make sure u don't use `javax.naming.ServiceUnavailableException`.